recipe/meta.yaml

```
@@ -30,6 +30,7 @@ requirements:
   build:
     - {{ compiler("c") }}
     - {{ compiler("cxx") }}
+    - {{ cdt('mesa-libgl-devel') }}
```

Needs to be constrained to Linux (Windows doesn't use these)

```suggestion
    - {{ cdt('mesa-libgl-devel') }}  # [linux]
```